### ISSUE TITLE
Voltage calibration for OSD broken?

### ISSUE THREAD

**Issue by MrMap** [FPV-Combat/Main_board_v2#8]
Tried the voltage calibration today, no joy.

HW 2.4
FW 2.5.0  

Used PUTTY, serial, 9600, Port 13 (as in com port list)
Board rebooted on FTDI connection but when I attempted to send $ I got no response whatsoever.
Calibration function broken in this FW?

**Comment by MrMap** [comment]
Today another reason to try the voltage calibration again, using another FTDi adapter. 
**SUCCESS - but slight change of procedure.**
HW 2.5
FW 2.5.0

Still using PUTTY, now com port 11.
This time I got a response to the $ in the terminal window: "Calib mode:"
No response when sending + or - (and return).
Restarted and entered Calib mode.
Tried pushing various keys and **when pushing "v" (for voltage?) the readings started to stream into the terminal window** and +/- worked for adjustment.

If this little change in procedure was published in documentation, well I missed it.
If not already documented it would be good to do that.
